doc/sphinx-guides/source/installation/config.rst

```diff
@@ -1858,6 +1858,31 @@ JSON files for `Creative Commons licenses <https://creativecommons.org/about/ccl
 
 .. _adding-custom-licenses:
 
+Adding Software Licenses
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+JSON files for software licenses are provided below.
+
+- :download:`licenseMIT.json <../../../../scripts/api/data/licenses/licenseMIT.json>`
+- :download:`licenseApache-2.0.json <../../../../scripts/api/data/licenses/licenseApache-2.0.json>`
+
+Contributing to the Collection of Standard Licenses Above
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you do not find the license JSON you need above, you are encouraged to contribute it to this documentation. Following the Dataverse 6.2 release, we have standardized on the following procedure:
+
+- Look for the license at https://spdx.org/licenses/
+- ``cd scripts/api/data/licenses``
+- Copy an existing license as a starting point.
+- Name your file using the SPDX identifier. For example, if the identifier is ``Apache-2.0``, you should name your file ``licenseApache-2.0.json``.
+- For the ``name`` field, use the "short identifier" from the SPDX landing page (e.g. ``Apache-2.0``).
+- For the ``description`` field, use the "full name" from the SPDX landing page (e.g. ``Apache License 2.0``).
+- For the ``uri`` field, we encourage you to use the same resource that DataCite uses, most of the times this will be the same than the first provided resource on SPDX, but please get in contact with us if you have any questions on the cominity group.
+- For the ``active`` field, put ``true``.
+- For the ``sortOrder`` field, put the next sequention number after checking previous files with ``grep sortOrder scripts/api/data/licenses/*``.
+
+Note that prior to Dataverse 6.2, various license above have been added that do not adhere perfectly with this procedure. For example, the ``name`` for the CC0 license is ``CC0 1.0`` (no dash) rather than ``CC0-1.0`` (with a dash). We are keeping the existing names for backward compatibility. For more on standarizing license configuration, see https://github.com/IQSS/dataverse/issues/8512
```

Comment:
If we don't use the DataCite URIs for these two, this should drop the "prior to Dataverse 6.2": Note that not all of the example licenses follow the recommended procedure above. These represent legacy cases or cases where community discussion has led the contributors to deviate from the recommendation. Given that internal consistency across Dataverse installations is important for interoperability and creating commons tools for exporting and parsing Dataverse metadata, we encourage you to use the provided .json files rather than creating new ones.